### ISSUE TITLE
[BUG] when click a checkbox in splade (data table) and have a rowLink it will redirect to row link route

### ISSUE THREAD

**Issue by zulfikar-ditya** [protonemedia/laravel-splade#309]
- Laravel Version: 9.19
- PHP Version: 8.0.12
- Splade JS Version (npm): 1.2.13
- Splade PHP Version (composer): 1.2.13.

### Description:
if we have a bulk action and row link in the same splade data table and we click the checkbox. it will redirect to row link route.

> ...Table.php
```
        $table
            ->withGlobalSearch(columns: [
                'name',
                'email',
                'phone',
                'website',
                'updated_at',
                'created_at',
            ])
            ->column('name', sortable: true, searchable: true)
            ->column('email', sortable: true, searchable: true)
            ->column('phone', sortable: true, searchable: true)
            ->column('website', sortable: true, searchable: true)
            ->column('updated_at', sortable: true, searchable: true)
            ->column('created_at', sortable: true, searchable: true)
            ->column('actions')
            ->rowLink(fn (Company $company) => route('master.company.show', $company))
            ->bulkAction(
                label: 'Delete data',
                each: fn (Company $project) => $project->delete(),
                confirm: 'Delete companies',
                confirmText: 'Are you sure you want to delete the companies?',
                confirmButton: 'Yes, delete all selected rows!',
                cancelButton: 'No, do not delete!',
            )
            ->defaultSortDesc('created_at');
```

**Comment by zulfikar-ditya** [comment]
#310

**Comment by pascalbaljet** [comment]
Will be released in v1.3 :)
